xbmc/video/windows/GUIWindowVideoBase.h

```diff
@@ -11,22 +11,18 @@
 #include "playlists/PlayListTypes.h"
 #include "video/VideoDatabase.h"
 #include "video/VideoThumbLoader.h"
+#include "video/guilib/VideoSelectAction.h"
 #include "windows/GUIMediaWindow.h"
 
-enum VideoSelectAction
+namespace
 {
-  SELECT_ACTION_CHOOSE          = 0,
-  SELECT_ACTION_PLAY_OR_RESUME,
-  SELECT_ACTION_RESUME,
-  SELECT_ACTION_INFO,
-  SELECT_ACTION_MORE,
-  SELECT_ACTION_PLAY,
-  SELECT_ACTION_PLAYPART,
-  SELECT_ACTION_QUEUE
-};
+class CVideoSelectActionProcessor;
+} // unnamed namespace
 
 class CGUIWindowVideoBase : public CGUIMediaWindow, public IBackgroundLoaderObserver
 {
+  friend class CVideoSelectActionProcessor;
```

Comment:
```suggestion
  friend class ::CVideoSelectActionProcessor;
```
Fixes build failures on Linux